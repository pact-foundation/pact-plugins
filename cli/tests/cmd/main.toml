bin.name = "pact-plugin"
status.code = 2
