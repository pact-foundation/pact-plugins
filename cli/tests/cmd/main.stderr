CLI utility for Pact plugins

Usage: pact-plugin [OPTIONS] <COMMAND>

Commands:
  list        List installed or available plugins
  env         Print out the Pact plugin environment config
  install     Install a plugin
  remove      Remove a plugin
  enable      Enable a plugin version
  disable     Disable a plugin version
  repository  Sub-commands for dealing with a plugin repository
  help        Print this message or the help of the given subcommand(s)

Options:
  -y, --yes      Automatically answer Yes for all prompts
  -d, --debug    Enable debug level logs
  -t, --trace    Enable trace level logs
  -v, --version  Print CLI version
  -h, --help     Print help
